idc/views.py

```diff
@@ -214,7 +214,6 @@ def quota_page(request):
 
 
 # Data exploration and cohort creation page
-@login_required
 def explore_data_page(request):
```

Comment:
This has to remain in until we get the go-ahead to actually be public.

Reply:
I have added this back. Thanks!